backend/tracim_backend/config.py

```diff
@@ -922,7 +923,9 @@ def _load_ldap_config(self) -> None:
         # TODO - G.M - 2019-04-05 - keep as parameters
         # or set it as constant,
         # see https://github.com/tracim/tracim/issues/1569
-        self.LDAP_USER_FILTER = "({}=%(login)s)".format(self.LDAP_LOGIN_ATTRIBUTE)
+        self.LDAP_USER_FILTER = "({}=%(login)s)".format(self.LDAP_USERNAME_ATTRIBUTE)
+        if self.EMAIL__REQUIRED:
+            self.LDAP_USER_FILTER = "({}=%(login)s)".format(self.LDAP_MAIL_ATTRIBUTE)
```

Comment:
as we discussed, create an issue about improving this part

Reply:
Done !

Reply:
We should also apply username and mail checks upon filling the info, because as it is now, invalid data could be passed to the DB

Reply:
#6180